api/.project/devstart.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "common/common"
+
+def startdb()
+  c = Common.new
+  env = c.load_env
+  cname = "#{env.namespace}-db"
+  c.run_inline %W{docker run --name #{cname}
+    -d
+    --network #{env.namespace}
+    -e MYSQL_ROOT_PASSWORD=root -e MYSQL_USER=workbench -e MYSQL_PASSWORD=workbench!pwd
+    -e MYSQL_DATABASE=workbench
+    library/mysql
+  }
+  at_exit { c.run_inline %W{docker rm -f #{cname}} }
+end
+
+def serve()
+  c = Common.new
+  env = c.load_env
+
+  c.run %W{docker network create #{env.namespace}}
```

Comment:
Adding the database to the same network makes the container name available as a host name.